### PR TITLE
simplifying eventlog tests, trying to remove thread reliance

### PR DESCRIPTION
**Pull request by shoeffner**
Removed lots of parts from the EventLog tests to make them slimmer and to attempt solving #20 #22 #45. We might need to dive into PowerMock to mock Thread.sleep in testPostEvent and testRunFinally, but I do hope that 5 seconds of optional timeout are enough for now.

### CODE REVIEW

**Comment by Faedrivin** [comment]
-> remove one test which was basically there for coverage (only calling the code, not actually checking results), hence the reduction by 0.02.

File: modules/eventlog/src/test/java/com/intuit/wasabi/eventlog/impl/EventLogImplTest.java [hunk 1]
```diff
@@ -24,10 +25,8 @@
 import org.mockito.Mockito;
 
 import java.lang.reflect.Field;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (iizrailevsky) [inline modules/eventlog/src/test/java/com/intuit/wasabi/eventlog/impl/EventLogImplTest.java]: Please specify classes used, instead of .*

**Comment by iizrailevsky** [comment]
@shoeffner Looks good! Apart from a small class import fix, shipit! 👍 :shipit:

File: modules/eventlog/src/test/java/com/intuit/wasabi/eventlog/impl/EventLogImplTest.java (hunk 1 above)
Comment (shoeffner) [inline modules/eventlog/src/test/java/com/intuit/wasabi/eventlog/impl/EventLogImplTest.java]: Good catch!

**Pull request merged**